move-mutator/tests/move-assets/simple_move_2_features/report.txt.spec-exp

```diff
@@ -111,5 +111,5 @@
       }
     ]
   },
-  "package_dir": "move-spec-testing/move-mutator/tests/move-assets/simple-move-2-features"
-}
+  "package_dir": "/Users/stoyankirov/projects/work/aptos/move-mutation-tools/move-mutator/tests/move-assets/simple_move_2_features"
```

Comment:
nit: I think this json value is not used when we run comparisons, so you can use something like this to avoid committing your info
```suggestion
  "package_dir": "move-mutation-tools/move-mutator/tests/move-assets/simple_move_2_features"
```